src/core/components/modal/Modal.vue

```diff
@@ -1,10 +1,10 @@
-<template>
-  <q-dialog :value="value" @input="$emit('input', $event)" @hide="$emit('hide', $event)" @show="$emit('show', $event)">
-    <div :class="containerClass">
+<template functional>
+  <q-dialog :value="props.value" v-on="listeners">
```

Comment:
Il y a une difference entre `v-on="listeners" et `v-on="$listeners"` ?

Reply:
`$listeners` est une propriété de l'instance. et les composant fonctionnels n'ont pas d'instance justement

Reply:
euh bonne question... j'ai trouvé listeners en creusant dans stackOverflow

Reply:
alors je crois que c'est pcq on est dans un composant fonctionnel et que on a pas accès à `this` -> du coup on utilise `listeners` (un peu comme `props`)